token_reject_transaction_unit_test.go

```diff
@@ -0,0 +1,318 @@
+//go:build all || unit
+// +build all unit
+
+package hedera
+
+/*-
+ *
+ * Hedera Go SDK
+ *
+ * Copyright (C) 2020 - 2024 Hedera Hashgraph, LLC
```

Comment:
I am not sure which option should be here (but copyright is a different topic):
- Copyright (C) 2020 - 2024 Hedera Hashgraph, LLC
- Copyright (C) 2024 Hedera Hashgraph, LLC (I use this as I believe saw such example in services)

Reply:
I guess that soon we would have to change that again.

Reply:
I can open a new PR for this.